app/views/partials/theme/simple/home/quick_search_form.liquid

```diff
@@ -0,0 +1,9 @@
+  <div class="mr-5">
+    <input class="form-input mr-2"
+      name="k"
+      id="k"
```

Comment:
@damcik Do you need this id? It will create global variable in js named `k`. If you really need this id, use different name `search-k` etc.

Reply:
i think name element should be enough, thanks for advice - id removed.